dataactbroker/README.md

```diff
@@ -801,6 +801,46 @@ List submissions for all agencies for which the current user is a member of. Opt
 }
 ```
 
```

Comment:
If you're adding the `list_agencies` route as well as your own, it might make sense to add the `list_sub_tier_agencies` route at the same time.